src/mmg3d/mmg3d1.c

```diff
@@ -694,11 +702,17 @@ MMG5_int MMG5_swptet(MMG5_pMesh mesh,MMG5_pSol met,double crit,double declic,
         }
 
         nconf = MMG5_chkswpgen(mesh,met,k,i,&ilist,list,crit,typchk);
-        if ( nconf<0 ) return -1;
+        if ( nconf<0 ){
+          MMG5_show_tet_location(mesh, pt, k);
+          return -1;
+        }
         else if ( nconf ) {
           ier = MMG5_swpgen(mesh,met,nconf,ilist,list,PROctree,typchk);
           if ( ier > 0 )  ns++;
-          else if ( ier < 0 ) return -1;
+          else if ( ier < 0 ){
+            MMG5_show_tet_location(mesh, pt, k);
```

Comment:
Please, keep this in debug mode only (`-d` command line option, `mesh->info.ddebug` variable inside the code) : 
```
if ( mesh->info.ddebug ) {
   MMG5_show_tet_location(mesh, pt, k);
}
```